charts/cass-operator/values.yaml

```diff
@@ -42,7 +42,7 @@ image:
   registryOverride:
 # -- References to secrets to use when pulling images. See:
 # https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
-# imagePullSecret: []
+imagePullSecret:
```

Comment:
We're changing from a secret reference to passing the name of the secret. Maybe a comment would help to explain how to pass it properly?

```suggestion
# imagePullSecret: secretName
```